types/kotlin/Error.kt

```diff
@@ -39,16 +39,3 @@ enum class Domain {
 enum class EventName {
     Error
 }
-
-enum class Name {
```

Comment:
is this going to break any existing code?

Reply:
kotlin is not used, the apps are using kotlin2

Reply:
so can we get rid of `kotlin` altogether? Maybe a followup task.

Reply:
https://github.com/matrix-org/matrix-analytics-events/issues/94